stringutil/decoder.go

```diff
@@ -102,5 +111,5 @@ func DecodeToASCII(s string, opts ...ASCIIDecodeOption) string {
 		return ""
 	}
 
-	return result
+	return strings.Trim(spaceRegex.ReplaceAllString(result, " "), " ")
```

Comment:
Not sure what this does: replace all mutli-spaces with single spaces, then trim spaces at the beginning and end?